crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -781,10 +781,9 @@ unsafe impl<'a, T: Resource> SystemParam for Res<'a, T> {
         world: UnsafeWorldCell,
     ) -> Result<(), SystemParamValidationError> {
         // SAFETY: Read-only access to resource metadata.
```

Comment:
I think you can remove the `SAFETY` comments here (and below for `ResMut`), since this check is entirely safe code now!